exercises/E-conditionals/exercise.js

```diff
@@ -8,6 +8,9 @@
 
 let name = "Daniel";
 let danielsRole = "mentor";
+if(danielsRole.localeCompare("mentor") === 0){
```

Comment:
Nice, great use of `localeCompare`. 
The use case for Daniel as a student is missing.